cmd/destroy/destroy.go

```diff
@@ -281,7 +305,7 @@ func (dc *destroyCommand) getDestroyer(ctx context.Context, opts *Options) (dest
 				return nil, err
 			}
 		}
-		runInRemote := !isRemote && (destroyImage != "" || opts.RunInRemote)
+		runInRemote := shouldRunInRemote(opts) && (destroyImage != "" || opts.RunInRemote)
```

Comment:
this operation is not needed. We just need to call `shouldRunInRemote()` an instantiate a remote or local deployer, the same approach you followed in the deploy command